charts/argo-cd/README.md

```diff
@@ -105,6 +105,11 @@ For full list of changes please check ArtifactHub [changelog].
 
 Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
 
+### 5.31.0
+The manifests are now using [`tini` as entrypoint][tinit], instead of `entrypoint.sh`. Until Argo CD v2.8, `entrypoint.sh` is retained for upgrade compatibility.
```

Comment:
`tini` instead of `tinit`? :thinking:

Reply:
typo 😇

Reply:
fixed at https://github.com/argoproj/argo-helm/pull/1991/commits/3412931f41066af20bb611a4fa0c15e8ab916402 🙋